codegenerator/cli/src/config_parsing/entity_parsing.rs

```diff
@@ -878,19 +878,28 @@ impl UserDefinedFieldType {
                              methods for referencing entities outlined in the docs. The entity \
                              being referenced in the array is '{}'.",
                             name
-                        ))?
+                        ))
+                    }
+                    //TODO: add support for these types
+                    //currently we would need to use explicid casts in the queries to make these
+                    //work https://github.com/porsager/postgres/pull/392
+                    Self::Single(GqlScalar::Boolean) => {
+                        Err(anyhow!("Arrays of booleans are not yet supported."))
+                    }
+                    Self::Single(GqlScalar::Timestamp) => {
+                        Err(anyhow!("Arrays of timestamps are not yet supported."))
```

Comment:
My tests revealed that arrays of boolean and arrays of timestamp are currently broken with our current queries for setting the values. The library requires explicit type casting. For now I'm adding a parse step here that will fail if someone tries to use one.